sass/tji/_variables.scss

```diff
@@ -4,6 +4,7 @@ $primaryBlue: #0B5D93;
 $primaryRed: #CE2727;
 	$secondaryRed: #9B1D1D;
 $primaryYellow: #FFFD00;
+$primaryGreen: #1D9B41;
```

Comment:
do we need to add another color? Can we just make this button fade to blue or secondaryRed so our color palette stays consistent?

Reply:
I didn't like using the secondary red for this, it's too dull. If everyone prefers we can have it fade to our blue though. I chose the green since this is a button to donate to TJI. Maybe we can push this code to our staging site and get everyone's thoughts on that color fade?

Reply:
sounds good, could we choose a color from the existing pallete tho, that the charts use.
It's defined in tji-datasets-explore.js

var COLOR_TJI_BLUE = '#0B5D93';
var COLOR_TJI_RED = '#CE2727';
var COLOR_TJI_DEEPBLUE = '#252939';
var COLOR_TJI_PURPLE = '#4D3B6B';
var COLOR_TJI_YELLOW = '#F1AB32';
var COLOR_TJI_TEAL = '#50E3C2';
var COLOR_TJI_DEEPRED = '#872729';
var COLOR_TJI_DEEPPURPLE = '#2D1752';

![screen shot 2018-07-07 at 1 14 08 pm](https://user-images.githubusercontent.com/3039125/42413522-f939d9cc-81e7-11e8-8ef6-af34124674fc.png)

Reply:
Sounds good Simi. I am going to give it the secondary red for now and we can discuss at the meeting further to see if anyone else has opinions.